frontend/javascripts/oxalis/view/right-menu/advanced_search_popover.js

```diff
@@ -128,14 +129,20 @@ export default class AdvancedSearchPopover<S: Object> extends React.PureComponen
                     style={{ width: "calc(100% - 100px)", ...additionalInputStyle }}
                     value={searchQuery}
                     placeholder="Enter your search keywords"
-                    onPressEnter={this.selectNextOption}
+                    onPressEnter={event => {
+                      if (event.shiftKey) {
+                        this.selectPreviousOption();
+                      } else {
+                        this.selectNextOption();
+                      }
+                    }}
                     onChange={evt =>
                       this.setState({ searchQuery: evt.target.value, currentPosition: null })
                     }
                     addonAfter={`${currentPosition + 1}/${numberOfAvailableOptions}`}
                     autoFocus
                   />
-                  <Tooltip title="Previous">
+                  <Tooltip title="Previous (shift+enter)">
```

Comment:
I added the shortcut in brackets so that it is easier to find for users. @daniel-wer do you think this is too much?

Reply:
I think it's very helpful as part of the tooltip :)